src/components/PersonItem.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Person } from '../types';
+import { Link, useParams } from 'react-router-dom';
+
+type Props = {
+  person: Person;
+};
+const PersonItem: React.FC<Props> = ({ person }) => {
+  const { slug } = useParams();
+
+  return (
+    <tr
+      data-cy="person"
+      className={person.slug === slug ? 'has-background-warning' : ''}
+    >
+      <td>
+        <Link
+          to={`${person.slug}`}
+          className={person.sex === 'f' ? 'has-text-danger' : ''}
+        >
+          {person.name}
+        </Link>
+      </td>
+
+      <td>{person.sex}</td>
+      <td>{person.born}</td>
+      <td>{person.died}</td>
+      <td>
+        {person.mother ? (
+          <Link to={person.mother.slug} className="has-text-danger">
+            {person.motherName}
+          </Link>
+        ) : (
+          person.motherName || '-'
+        )}
+      </td>
+      <td>
+        {person.father ? (
+          <Link to={person.father.slug}>{person.father.name}</Link>
```

Comment:
you use here father.name but in previous line - motherName. I may be wrong, but I think it's better to use similar notation for these outputs